model/host/db.go

```diff
@@ -128,6 +134,46 @@ func AllIdleEphemeral() ([]Host, error) {
 	return Find(query)
 }
 
+func IdleEphemeralGroupedByDistroId(creationTimeAsc bool) ([]HostsByDistro, error) {
+	var hostsByDistro []HostsByDistro
+	sortOrder := -1
+	if creationTimeAsc {
+		sortOrder = 1
+	}
```

Comment:
I'd be fine if we just always sorted it in the direction that we wanted. I don't think there's a viable use-case to terminate newest hosts

Reply:
Removed.